extensions/wikia/AdEngine/js/spec/lookup/prebid/adaptersPricesTracker.spec.js

```diff
@@ -8,7 +8,7 @@ describe('ext.wikia.adEngine.lookup.prebid.adaptersPricesTracker', function () {
 	function getModule() {
 		return modules['ext.wikia.adEngine.lookup.prebid.adaptersPricesTracker'](
 			mocks.adaptersRegistry,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the declaration of "mocks" before this usage. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)